app/controlplane/pkg/metrics/prometheus/models.go

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheuscollector
+
+// ChainloopMetricsGatherer is an interface for a gatherer that gathers metrics for Chainloop prometheus collector
+type ChainloopMetricsGatherer interface {
```

Comment:
I'd not call this file `models`. `Models` is not idiomatic in Go.

I'd just add them to the collector file or a prometheus.go, but either way splitting it looks like an early optimization.